feg/gateway/services/session_proxy/credit_control/gx/gx_client.go

```diff
@@ -236,14 +236,13 @@ func (gxClient *GxClient) createCreditControlMessage(
 	m.NewAVP(avp.IPCANType, avp.Mbit|avp.Vbit, diameter.Vendor3GPP, datatype.Enumerated(request.IPCANType))
 	m.NewAVP(avp.RATType, avp.Vbit, diameter.Vendor3GPP, datatype.Enumerated(request.RATType))
 
+	// IPv4
 	if ip := net.ParseIP(request.IPAddr); ipNotZeros(ip) {
 		if ipV4 := ip.To4(); ipV4 != nil {
 			m.NewAVP(avp.FramedIPAddress, avp.Mbit, 0, datatype.IPv4(ipV4))
 		} else if gxClient.framedIpv4AddrRequired {
 			defaultIp := getDefaultFramedIpv4Addr()
 			m.NewAVP(avp.FramedIPAddress, avp.Mbit, 0, datatype.IPv4(defaultIp))
```

Comment:
why not to leave else if ... for ipv6, but add a condition that it's not already set:

1) move the added IPV6 block above IPV4
2)add var ipV6Prefix net.IP
3) change the condition to:
} else ipV6Prefix == nil {
     if ipV6 := ip.To16(); ipV6 != nil {	
			m.NewAVP(avp.FramedIPv6Prefix, avp.Mbit, 0, datatype.OctetString(ipV6))	
		} 
     }